lib/Doctrine/ODM/PHPCR/Translation/TranslationStrategy/AttributeTranslationStrategy.php

```diff
@@ -173,9 +173,9 @@ public function removeAllTranslations($document, NodeInterface $node, ClassMetad
     public function getLocalesFor($document, NodeInterface $node, ClassMetadata $metadata)
     {
         $locales = array();
-        foreach ($node->getProperties("*{$this->prefix}*") as $prop) {
+        foreach ($node->getProperties($this->prefix . ':*') as $prop) {
```

Comment:
Note that I changed the pattern for `getProperties`. It was previously allowing text before the prefix and not requiring the prefix delimited `:`.  Is that correct?